ryot/src/appearances/appearances.proto

```diff
@@ -99,7 +99,7 @@ message SpriteInfo {
 	optional uint32 pattern_height = 2;
 	optional uint32 pattern_depth = 3;
 	optional uint32 layers = 4;
-	repeated uint32 sprite_id = 5;
```

Comment:
Noice, I think I'll add a linear ticket to review this proto and put better and more intuitive names on it.

Reply:
https://linear.app/ryot/issue/RYO-60/review-appearancesproto-names